tensorflow/contrib/rnn/python/kernel_tests/rnn_cell_test.py

```diff
@@ -1554,6 +1554,16 @@ def testMinimalRNNCellEndToEnd(self):
       self.assertEqual(len(outputs), batch)
       self.assertEqual(len(state), batch)
 
+  def testWrapperWithKerasLSTMCellGetInitialState(self):
```

Comment:
I think you also need to test `cell.call`

And I am not sure but I think it would require some changes to adapt to the keras state format (similarly to here: https://github.com/tensorflow/tensorflow/blob/7cf8899b2194c3049e65dc136751b2bfa6d79f5d/tensorflow/python/ops/rnn.py#L824)